docs/en/modules/net.md

```diff
@@ -364,6 +396,44 @@ none
 #### See also
 [`net.socket:hold()`](#netsockethold)
 
+# net.udpsocket Module
+
+## net.udpsocket:close()
+
+Closes UDP socket.
+
+The syntax and functional identical to [`net.socket:close()`](#netsocketclose).
+
+## net.udpsocket:listen()
+
+Listen on port from IP address.
+
+The syntax and functional similar to [`net.server:listen()`](#netserverlisten), but callback parameter is not provided.
+
+## net.udpsocket:on()
+
+Register callback functions for specific events.
+
+The syntax and functional similar to [`net.socket:on()`](#netsocketon), only "received", "sent" and "dns" events is valid.
+
+**`received` callback have `port` and `ip` before `data` argument.**
```

Comment:
Don't forget to update this when the args are moved. :)

Overall we should probably expand on the `net` docs, they've always been a bit light... but that's for later.